war/pom.xml

```diff
@@ -393,7 +393,35 @@ THE SOFTWARE.
                 <artifactItem>
                   <groupId>org.jenkins-ci.plugins</groupId>
                   <artifactId>junit</artifactId>
-                  <version>1.6</version>
+                  <version>1.26.1</version>
+                  <type>hpi</type>
+                </artifactItem>
+                <artifactItem>
+                  <!-- dependency of junit -->
```

Comment:
Yikes. Maybe we should start deleting detached plugins that have been detached for a long time—this could get out of hand. In this case, `junit` was detached nearly five years ago.

Reply:
Or – and this might sound crazy – it should have been an optional dependency for optional extensions in the first place 😄